python_modules/libraries/dagster-dbt/dagster_dbt/include/scaffold/definitions.py.jinja

```diff
@@ -1,42 +1,16 @@
 import os
-from pathlib import Path
 
-from dagster import Definitions, OpExecutionContext
-from dagster_dbt import DbtCliResource, build_schedule_from_dbt_selection, dbt_assets
+from dagster import Definitions
+from dagster_dbt import DbtCliResource
 
-{% if use_dbt_project_package_data_dir -%}
-# We expect the dbt project to be installed as package data.
-# For details, see https://docs.python.org/3/distutils/setupscript.html#installing-package-data.
-{%- endif %}
-dbt_project_dir = Path(__file__).joinpath({{ dbt_project_dir_relative_path_parts | join(', ')}}).resolve()
-dbt = DbtCliResource(project_dir=os.fspath(dbt_project_dir))
-
-# If DAGSTER_DBT_PARSE_PROJECT_ON_LOAD is set, a manifest will be created at runtime.
-# Otherwise, we expect a manifest to be present in the project's target directory.
-if os.getenv("DAGSTER_DBT_PARSE_PROJECT_ON_LOAD"):
-    dbt_parse_invocation = dbt.cli([{{ dbt_parse_command | join(', ')}}], manifest={}).wait()
-    dbt_manifest_path = dbt_parse_invocation.target_path.joinpath("manifest.json")
-else:
-    dbt_manifest_path = dbt_project_dir.joinpath("target", "manifest.json")
-
-@dbt_assets(manifest=dbt_manifest_path)
-def {{ dbt_assets_name }}(context: OpExecutionContext, dbt: DbtCliResource):
-    yield from dbt.cli(["build"], context=context).stream()
-
-
-schedules = [
-    build_schedule_from_dbt_selection(
-        [{{ dbt_assets_name }}],
-        job_name="materialize_dbt_models",
-        cron_schedule="0 0 * * *",
-        dbt_select="fqn:*",
-    )
-]
+from .assets import jaffle_shop_dbt_assets
```

Comment:
should this be {{ dbt_assets_name }}?

Reply:
I'm surprised that the tests missed this... thanks for pointing this out, will look into it

Reply:
Should be addressed in https://github.com/dagster-io/dagster/pull/15756